src/rabbit_federation_mgmt.erl

```diff
@@ -26,17 +26,20 @@
 
 -include_lib("rabbitmq_management_agent/include/rabbit_mgmt_records.hrl").
 
-dispatcher() -> [{"/federation-links",        ?MODULE, []},
-                 {"/federation-links/:vhost", ?MODULE, []}].
+dispatcher() -> [{"/federation-links",        ?MODULE, [all]},
+                 {"/federation-links/:vhost", ?MODULE, [all]},
+                 {"/federation-down-links",        ?MODULE, [down]},
```

Comment:
OK, maybe there is is a vhost named `down` somewhere and it's a bad idea. I have habits back from my Rails days 🚄 .